api/app/src/Service/UserRegistrationService.php

```diff
@@ -146,12 +146,8 @@ public function saveUserAndClient(User $user, Client $client)
             $this->em->persist($user);
             $this->em->flush();
 
-            // Add the user to the client an save it
-            /* @var Client $client */
+            // Add the user to the client then save
             $client->addUser($user);
-            $user->setCreatedBy($user);
```

Comment:
This was initially added due to wanting to track the route a user had taken. We now have something better in place, as leaving this here for self-registered users would leave them out of scope for the cleanup job I've removed this. Will adjust data so all self referencing created_by are removed.